samples/Mvc/Mvc.Server/Startup.cs

```diff
@@ -43,6 +43,8 @@ public class Startup {
                 .AllowAuthorizationCodeFlow()
                 .AllowRefreshTokenFlow()
 
+                .AddEphemeralSigningKey()
```

Comment:
This line is still there? :smile: We really don't want to generate 2 RSA keys at runtime (it takes time and there's no point doing that).